Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.py

```diff
@@ -0,0 +1,831 @@
+
+    command_results = CommandResults(
+        outputs_prefix=f'{INTEGRATION_OUTPUTS_BASE_PATH}.Attachment',
+        outputs_key_field='ID',
+        outputs=outputs,
+        readable_output=tableToMarkdown('Attachments', outputs, headers=hr_header, removeNull=True)
+    )
+
+    if not download_file:
+        return command_results
+
+    files = []
+    for attachment in res:
+        attachment_url = attachment.get('url')
+        file_response = client.get_file(attachment_url)
+        file_name = attachment_url.split('/')[-1]
+        i = 1
+        while file_name in files:
+            name, ext = file_name.split('.')
+            file_name = f"{name}_{i}.{ext}"
+            i += 1
+        with open(file_name, 'wb') as file:
+            file.write(file_response.content)
+            files.append(file.name)
+    zip_data_buffer = io.BytesIO()
+    with zipfile.ZipFile(zip_data_buffer, 'w') as zipf:
+        for file in files:
+            zipf.write(file)
+            os.remove(file)
+
+    data = zip_data_buffer.getvalue()
+    file_result = fileResult('ObjectAttachments.zip', data, EntryType.ENTRY_INFO_FILE)
+    return [file_result, command_results]
+
+
+def jira_asset_attachment_remove_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Removes a Jira asset attachment.
+    :param client: Client object for performing API requests.
+    :param args: A dictionary of command arguments:
+        - 'id': The ID of the attachment to remove.
+    :return: A CommandResults object containing the attachment as output, and a confirmation message.
+    """
+    attachment_id = args.get('id')
+
+    try:
+        res = client.remove_file(attachment_id)
+    except DemistoException as e:
+        if e.res.status_code == 404:
+            return CommandResults(readable_output=f'Attachment with id: {attachment_id} does not exist')
+        else:
+            raise e
+
+    outputs = convert_keys_to_pascal([res], key_mapping={'id': 'ID'})
+    return CommandResults(
+        outputs_prefix=f'{INTEGRATION_OUTPUTS_BASE_PATH}.Attachment',
+        outputs_key_field='ID',
+        outputs=outputs,
+        readable_output=f'Attachment with id {res.get("id")} was successfully deleted'
+    )
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    command = demisto.command()
+    args = demisto.args()
+    params = demisto.params()
+    api_key = params.get('credentials', {}).get('password', '')
+    server_url = params.get('url')
+    verify_certificate = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    base_url = server_url
+
+    try:
+        base_url = ON_PREM_URL_STRUCTURE.format(server_url)
+        client = Client(base_url=base_url, verify=verify_certificate, proxy=proxy, api_key=api_key)
+
+        if command == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            return_results(result)
```

Comment:
move return_results to the end since its common to all cases
extra points for converting to a dict like
`{ 'command_name' : comand_func }.get(command)()`

Reply:
done